prompt.py

```diff
@@ -933,6 +941,32 @@ def run(self):
                 traceback.print_exc()
 
 
+def check_privatenet():
+    """ Check if privatenet is running, and if container is same as the chain file """
+    host = "http://127.0.0.1:30333"
+    rpc_settings.setup([host])
+    client = RPCClient()
+    version = client.get_version()
+    # print("version", version)
+    if not version:
+        logger.error("Error: private network doesn't seem to be running")
```

Comment:
"or RPC server is not enabled."

In continuation of the previous comment. I don't think we should assume everybody to use the CoZ or your docker container. Some might have followed the docs.neo.org steps and don't have the RPC server enabled.

Reply:
done